workflow_tests/testCreateContent.js

```diff
@@ -0,0 +1,301 @@
+        },
+      };
+  }
+}
+
+function expectedSection(sectionCase) {
+  switch (sectionCase) {
+    case "none":
+      return {
+        language: "en-US",
+        preferences: null,
+        template: "blog-post",
+        templateParams: {
+          contentLength: 1000,
+          language: "en-US",
+          sections: "[]",
+          summary: "a blog post about how important bees are to my honey farm",
+        },
+      };
+    case "one":
+      return {
+        language: "en-US",
+        preferences: null,
+        template: "blog-post",
+        templateParams: {
+          contentLength: 1000,
+          language: "en-US",
+          sections: "['Bees and me']",
+          summary: "a blog post about how important bees are to my honey farm",
+        },
+      };
+    case "multiple":
+      return {
+        language: "en-US",
+        preferences: null,
+        template: "blog-post",
+        templateParams: {
+          contentLength: 1000,
+          language: "en-US",
+          sections: "['Bees and me', 'Honey for you', 'Conclusion']",
+          summary: "a blog post about how important bees are to my honey farm",
+        },
+      };
+    case "nonList":
+      return {
+        language: "en-US",
+        preferences: null,
+        template: "blog-post",
+        templateParams: {
+          contentLength: 1000,
+          language: "en-US",
+          sections: "1",
+          summary: "a blog post about how important bees are to my honey farm",
+        },
+      };
+    case "nonStringElements":
+      return {
+        language: "en-US",
+        preferences: null,
+        template: "blog-post",
+        templateParams: {
+          contentLength: 1000,
+          language: "en-US",
+          sections: "[1, 2]",
+          summary: "a blog post about how important bees are to my honey farm",
+        },
+      };
+  }
+}
+
+function expectResponse(expect, createContentObj, expected) {
+  expect(createContentObj.language).to.equal(expected["language"]);
+  expect(createContentObj.template).to.equal(expected["template"]);
+  let templateParams = createContentObj.templateParams;
+  expect(templateParams.contentLength).to.equal(templateParams["contentLength"]);
+  expect(templateParams.language).to.equal(templateParams["language"]);
+  expect(templateParams.sections).to.equal(templateParams["sections"]);
+  expect(templateParams.summary).to.equal(templateParams["summary"]);
+}
+
+(function(root, factory) {
+    if (typeof define === 'function' && define.amd) {
+      // AMD.
+      define(['expect.js', process.cwd()+'/src/index'], factory);
+    } else if (typeof module === 'object' && module.exports) {
+      // CommonJS-like environments that support module.exports, like Node.
+      factory(require('expect.js'), require(process.cwd()+'/src/index'));
+    } else {
+      // Browser globals (root is window)
+      factory(root.expect, root.LiltNode);
+    }
+}(this, function(expect, LiltNode) {
```

Comment:
why do we need to do this thing?

Reply:
it doesn't import LiltNode correctly without it and this is how the autogenerated test code imports the module, don't think it's all necessary but thought it was good to maintain the same pattern as the autogenerated tests